File: app/src/main/java/acm/androidfitnesseexample/DefaultActivity.java
package acm.androidfitnesseexample;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

public class DefaultActivity extends Activity implements MessageInterface {

    private TextView mMessage;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_default);
        mMessage = findViewById(R.id.message);
    }

    @Override
    public void showMessage(String text) {
        mMessage.setText(text);
    }
}

File: app/src/main/java/acm/androidfitnesseexample/DefaultActivity.kt
package acm.androidfitnesseexample

import android.app.Activity
import android.os.Bundle
import android.widget.TextView

class DefaultActivity : Activity(), MessageInterface {

    private var mMessage: TextView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_default)
        mMessage = findViewById(R.id.message)
    }

    override fun showMessage(text: String) {
        mMessage!!.text = text
    }
}